src/config/backpack/operations/update.php

```diff
@@ -21,6 +21,9 @@
     // when the page loads, put the cursor on the first input?
     'autoFocusOnFirstField' => true,
 
+    // when enabled, the update button will open a modal form instead of the edit page
+    'updateButtonWithModalForm' => true,
```

Comment:
No no no no. Bad Pedro! Why would we change the default behaviour, with something we're not even 100% sure works all of the time, for all fields? No.

```suggestion
    'updateButtonWithModalForm' => false,
```